deploy/helm/kubecf/templates/db-seeder.yaml

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
For consistency, we should name this file `database_seeder.yaml`.

Reply:
This file went away with https://github.com/SUSE/kubecf/pull/393/commits/4328e181004a5668963f03d3b7c5fdc8742c4441.